openverse_catalog/dags/providers/provider_api_scripts/jamendo.py

```diff
@@ -114,7 +115,22 @@ def _get_batch_json(
         return None
     else:
         results = response_json.get("results")
-        return results
+        return force_duplicates(results)
+
+
+# Takes some results and forcibly adds some duplicates to them
+def force_duplicates(results):
```

Comment:
Do you mean all of the `force_duplicates` function? I didn't look into it much, but it seems like it should be a part of tests, not the script itself...
Should I remove this function and logs above it?

Reply:
I assumed all of the changes to this file were by mistake. I believe the `force_duplicates` function was actually a patch I provided a while ago in a different PR where I had made some changes to the `loader` code -- it was a function I was using to manually force there to be some duplicate results, so we could test that the loader successfully removed them. (I may be remembering the details slightly wrong 😄) I didn't intend for it to get merged -- looks like it might've accidentally got into this changeset, unless I'm missing something?